repos/keg-core/core/base/utils/platform/isIframe.js

```diff
@@ -0,0 +1,11 @@
+import { getWindow } from './getWindow'
+
+/**
+ * @returns {Boolean} true if this react app is in a web environment and also rendered inside of an iframe
+ */
+export const isIframe = () => {
+  const win = getWindow()
+  return win
+    ? win.location !== win.parent.location
```

Comment:
While this check would work the majority of the time, it's technically not accurate.
  * Because both the location and the iframe could have the same location
  * A better check would be `window === window.parent`

Reply:
Updated